### PR TITLE
Add url as admin

### PR DESCRIPTION
**Pull request by jHelsing**
Feature 30 is now implemented and it is possible to add an image URL to both doctors and nurses. If a nurse or doctor has an imageUrl, they are displayed instead of the random image in the lists. It is also displayed in doctor details. Currently Random Doctor and Ugly Nurse has the profile image connected to themselves.

### CODE REVIEW

File: src/models/nurse.ts [hunk 1]
```diff
@@ -5,4 +5,5 @@ export interface Nurse {
   name: string;
   dateOfBirth: Date;
   roomRef: DocumentReference;
+  imageUrl?: String;
```
Comment (davidemdot) [inline src/models/nurse.ts]: It should be `string`, shouldn't it?

File: src/pages/doctor-detail/doctor-detail.html [hunk 2]
```diff
@@ -13,6 +13,7 @@
   <ion-list>
     <ion-list-header>Basic Info</ion-list-header>
     <ion-card>
+      <img *ngIf="doctor.imageUrl" src="{{doctor.imageUrl}}">
```
Comment (davidemdot) [inline src/pages/doctor-detail/doctor-detail.html:16]: Maybe could we set a maximum size for the picture (here and in `nurse-detail`)?

Comment (jHelsing) [reply src/pages/doctor-detail/doctor-detail.html:16]: I tested with a 4000-ish wide image, and it shrinked until it's width fitted inside the card, so I do not think that this is a problem, unless we start uploading things like screenshots and similar stuff

File: src/models/nurse.ts (hunk 1 above)
Comment (jHelsing) [reply src/models/nurse.ts]: Corrected

File: src/pages/doctor-detail/doctor-detail.html (hunk 2 above)
Comment (davidemdot) [reply src/pages/doctor-detail/doctor-detail.html:16]: Good!

**Review by davidemdot — APPROVED** [review]

**Review by lagebergman — APPROVED** [review]

**Pull request merged**